extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/ParamConverterProvidersSupplier.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.resteasy.reactive.server.runtime;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class ParamConverterProvidersSupplier implements Supplier<List<ParamConverterProvidersSupplier.Entry>> {
+
+    @Override
+    public List<ParamConverterProvidersSupplier.Entry> get() {
```

Comment:
Why not return the list of ParamConverterProviders directly?

Reply:
We could, but that would expose the template to internals that could potentially change.